force-app/main/default/lwc/accountMap/__tests__/accountMap.test.js

```diff
@@ -0,0 +1,80 @@
+import { createElement } from 'lwc';
+import AccountMap from 'c/accountMap';
+import { getRecord } from 'lightning/uiRecordApi';
+import { registerLdsTestWireAdapter } from '@salesforce/sfdx-lwc-jest';
+
+// Realistic data with an accounts address details
+const mockGetRecord = require('./data/getRecord.json');
+
+// Register as a LDS wire adapter. Some tests verify the provisioned values trigger desired behavior.
+const getRecordAdapter = registerLdsTestWireAdapter(getRecord);
+describe('c-account-map', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('displays a lightning-map when the wire adaptor returns records', () => {
```

Comment:
This should be `displays a lightning-map when wire adaptor returns an account record with billing street data`. That would describe this test best. Also the `getRecord` wire adapter - not `adaptor` - always returns a single record.